samples/Microsoft.AspNet.SignalR.Samples/Scripts/hubs.js

```diff
@@ -1,5 +1,5 @@
 /*!
- * ASP.NET SignalR JavaScript Library v2.2.0-pre
```

Comment:
I must have missed this in a previous commit - I probably didn't stage the file or something to this effect.